.flake8

```diff
@@ -1,7 +1,3 @@
 [flake8]
 max-line-length = 120
 inline-quotes = double
-
-# We use 'black' for coding style; these next two warnings are not PEP-8
-# compliant.
-ignore = E203, W503
```

Comment:
why is this not relevant anymore? Even if it's added automatically by the actions, it'll confuse things when running flake8 locally, no?

Reply:
I no longer get any of these warnings on repositories. So I guess this is fixed in flake8 or black :)

It isn't automatically added; also locally you no longer need it.

Reply:
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8 Would suggest that E203 is still necessary, at least

Reply:
Good call! Fixed :)